admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.ts

```diff
@@ -186,9 +214,24 @@ export default class ProductPartialUpdater {
    *
    * @private
    */
-  private updateSubmitButtonState(): void {
+  private updateFooterButtonStates(): void {
     const updatedData = this.getUpdatedFormData();
-    this.$productFormSubmitButton.prop('disabled', updatedData === null);
+
+    if (updatedData === null) {
+      this.$productFormSubmitButton.prop('disabled', true);
+      this.$productFormCancelButton.addClass('disabled');
+      this.$productFormGoToCatalogButton.removeClass('disabled');
+      this.$productFormPreviewButton.removeClass('disabled');
+      this.$productFormDuplicateButton.removeClass('disabled');
+      this.$productFormNewProductButton.removeClass('disabled');
```

Comment:
Seems like with this the button will be displayed as disabled but not actually disabled, is it on purpose to allow having tooltips? Why not but then some additional JS must be added to mimic the disabled behavior and avoid an action to be triggered when the button is clicked